Include/internal/pycore_object.h

```diff
@@ -890,6 +890,8 @@ extern bool _PyObject_TryGetInstanceAttribute(PyObject *obj, PyObject *name,
                                               PyObject **attr);
 extern PyObject *_PyType_LookupRefAndVersion(PyTypeObject *, PyObject *,
                                              unsigned int *);
+ extern unsigned int
```

Comment:
```suggestion
 extern uint32_t
```
`int` returns imply a very limited range of values, usually only 0/-1.  Please use C99 int types when returning actual data, rather just an error code.

Should the comment explaining what it does go here, rather than on the implementation?
It seems to me that "what" comments go in headers, and "how" comments in the C code.

Reply:
I've moved the comment to the header file. The return value is `unsigned int` because `tp_version_tag` (in PyTypeObject) is an `unsigned int` and all the other `_PyType_Lookup` variants store the version tag as an `unsigned int`.